src/librustc/ty/constness.rs

```diff
@@ -36,12 +36,47 @@ impl<'tcx> TyCtxt<'tcx> {
         }
     }
 
+    /// Returns `true` if the `def_id` refers to an intrisic which we've whitelisted
+    /// for being called from stable `const fn`s (`min_const_fn`).
+    ///
+    /// Adding more intrinsics requires sign-off from @rust-lang/lang.
```

Comment:
I assume the other whitelist also requires rust-lang/lang sign-off?

Please also explain the relationship between the two whitelists.

Reply:
The other list is just unstable stuff, adding more unstable const stuff has so far only required lang sign off if there's no RFC justifying the experimentation

Reply:
How's that unstable, isn't that used in `const` and `static` bodies?

Reply:
ok, so the other list is the *full* list, unstable and stable, and this list is just the stable list. (The comments say so now, too)

Reply:
Oh I see, that makes sense. Thanks!